tests/test_ocr.py

```diff
@@ -1,10 +1,9 @@
 import unittest
-
-from segmentation import ContourSegmenter
-from feature_extraction import SimpleFeatureExtractor
-from files import ImageFile
-from classification import KNNClassifier
-from ocr import OCR, reconstruct_chars
+from simpleocr.segmentation import ContourSegmenter
+from simpleocr import SimpleFeatureExtractor
+from simpleocr import ImageFile
+from simpleocr import KNNClassifier
+from simpleocr import OCR, reconstruct_chars
```

Comment:
Please be consistent on importing from `simpleocr` (i.e.: `__init__` vs `simpleocr.X)` vs the other tests